index.html

```diff
@@ -33,5 +33,220 @@ <h1 class="title1">Hey there. I'm Francisco<br><span class="title2">I'm a softwa
+        </div>
+    </section>
+    <br>
+    <br>
+    <section>
+        <div class="div1">
+            <div>
+                <img class="paper-bird1" src="images/SVG/illustration 1- about me-mobile.svg" alt="paper-bird1">
+                <h2 class="color1">About me</h2>
+                <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.</p>
+            </div>
+        </div>
+        <br>
+        <div class="div1">
+            <div class="div3">
+                <img class="paper-bird2" src="images/SVG/Frame.svg" alt="paper-bird2">
+                <a class="button" href="#">
+                    <button class="seeproject">Get My Resume</button>
+                </a>
+            </div>
+        </div>
+    </section>
+    <br>
+    <br>
+    <section>
+        <div class="container2">
+            <div class="item2">
+                <img class="icon-skills" src="images/SVG/icon-languages.svg" alt="icon-skills">
+            </div>
+            <div class="item2">
+                <h3>Languages</h3>
+            </div>
+            <div class="item2">
+                <ul class="skills2">
+                    <li class="skills3">JavaScript</li>
+                    <li class="skills3">Ruby</li>
+                    <li class="skills3">HTML</li>
+                    <li class="skills3">CSS</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <br>
+    <section>
+        <div class="container2">
+            <div class="item2">
+                <img class="icon-skills" src="images/SVG/icon-frameworks.svg" alt="icon-skills">
+            </div>
+            <div class="item2">
+                <h3>Frameworks</h3>
+            </div>
+            <div class="item2">
+                <ul class="skills2">
+                    <li class="skills3">Bootstrap</li>
+                    <li class="skills3">Ruby on rails</li>
+                    <li class="skills3">RSpec</li>
+                    <li class="skills3">Capybara</li>
+                    <li class="skills3">Selenium</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <br>
+    <section>
+        <div class="container2">
+            <div class="item2">
+                <img class="icon-skills" src="images/SVG/icon-skills.svg" alt="icon-skills">
+            </div>
+            <div class="item2">
+                <h3>Skills</h3>
+            </div>
+            <div class="item2">
+                <ul class="skills2">
+                    <li class="skills3">Codekit</li>
+                    <li class="skills3">GitHub</li>
+                    <li class="skills3">Codepen</li>
+                    <li class="skills3">Gitlab</li>
+                    <li class="skills3">Terminal</li>
+                </ul>
+            </div>
+        </div>
+    </section>
```

Comment:
I think you did a great job here, however, there are some things to improve here. :+1: :smiley: :smiley: 
- This block of code should be separated into two sections. One for "Works" and another one for "About me". Kindly wrap these sections inside `section` tags. Any other `section` tag should be removed. Every work or skill card should be inside a `div` tag, not a section tag. 
![image](https://user-images.githubusercontent.com/98363075/198801059-6d2356ba-b70b-4ff1-9703-c7bdb00552ed.png)

- The images in the about section should be added as background images using CSS. Decorations are always background images, not `img` tags. :nerd_face: :nerd_face:  Kindly change this. 
![image](https://user-images.githubusercontent.com/98363075/198800802-c9fad4df-3832-431c-8bdb-e7dd1b9da908.png)